cylc/flow/scripts/install.py

```diff
@@ -18,11 +18,8 @@
 
 """cylc install [OPTIONS] ARGS
 
-Install a new workflow.
-
-Install the name REG. The workflow server program can then be started, stopped,
-and targeted by name REG. (Note that "cylc play" can also install workflows on
-the fly).
+Install a new workflow. The workflow can then be started, stopped, and targeted
+by name.
```

Comment:
The first line is used for the command summary i.e. `cylc help all`:

```suggestion
Install a new workflow.

The workflow can then be started, stopped, and targeted by name.
```

Also just a good system for documenting things:

```
Short concise description.

Longer explanation that can span multiple paragraphs, contain examples and extra information.

The short description should be easily skimable. Even if the full meaning is not immediately apparent to all
users the general scope of its functionality should be obvious enough that someone can quickly decide whether
this is information relevant to their search.
```